### PR TITLE
IDE pro references removed. Some bad indentations inside code blocks+typos

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
- Updating the Portenta H7 tutorials:
 IDE pro references removed. Some bad indentations inside code blocks+typos

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-h7/tutorials/dual-core-processing/content.md [hunk 1]
```diff
@@ -192,4 +192,4 @@ This tutorial introduces the idea of dual core processing and illustrates the co
 
 ### Next Steps
 
-- Proceed with the next tutorial "Setting Up a Wi-Fi Access Point" to learn how to make use of the built-in Wi-Fi module and configure your Portenta H7 as a Wi-Fi access point.
+- Proceed with the next tutorial [Setting Up a Wi-Fi Access Point](https://docs.arduino.cc/tutorials/portenta-h7/wifi-access-point) to learn how to make use of the built-in Wi-Fi module and configure your Portenta H7 as a Wi-Fi access point.
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-h7/tutorials/dual-core-processing/content.md:195]: @jcarolinares we should make this into a relative path link. I think `tutorials/portenta-h7/wifi-access-point` is the correct one

File: content/hardware/04.pro/boards/portenta-h7/tutorials/setting-up-portenta/content.md [hunk 2]
```diff
@@ -96,25 +92,23 @@ void loop() {
 
 For Portenta H7 LED_BUILTIN represents the built-in RGB LED on the board in green color.
 
-**Note:** The individual colors of the built-in RGB LED can be accessed and controlled separately. In the tutorial "Dual Core Processing" you will learn how to control the LED to light it in different colors
+**Note:** The individual colors of the built-in RGB LED can be accessed and controlled separately. In the tutorial [Dual core processing](https://docs.arduino.cc/tutorials/portenta-h7/dual-core-processing) you will learn how to control the LED to light it in different colors
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-h7/tutorials/setting-up-portenta/content.md:95]: Same here with the relative links ⭐

**Review by marqdevx — APPROVED** [review]
LGTM

**Review by jhansson-ard — APPROVED** [review]
Links seems to be working in the preview! Thanks for fixing it!

**Comment by jcarolinares** [comment]
Thanks @jhansson-ard and @marqdevx for the review and fixes!

Merging!

**Pull request merged**